packages/cli/src/commands/init.ts

```diff
@@ -33,20 +51,68 @@ async function gitHubOAuth(): Promise<string> {
     return await serverResponse(httpServer);
   } catch (error) {
     // eslint-disable-next-line no-console
-    console.log(error);
+    console.error(error);
     return null;
   } finally {
     httpServer.close();
   }
+}
 
-  function serverResponse(server: HttpServer): Promise<string> {
-    return new Promise((resolve) => {
-      const interval = setInterval(() => {
-        if (server.token) {
-          clearInterval(interval);
-          resolve(server.token);
-        }
-      }, 300);
-    });
-  }
+function serverResponse(server: HttpServer): Promise<string> {
+  return new Promise((resolve) => {
+    const interval = setInterval(() => {
+      if (server.token) {
+        clearInterval(interval);
+        resolve(server.token);
+      }
+    }, 300);
+  });
+}
+
+async function createOrganization(config: Configstore, organizationName: string): Promise<IOrganizationDTO> {
+  return (
+    await axios.post(
+      API_CREATE_ORGANIZATION_URL,
+      { name: organizationName },
+      {
+        headers: {
+          authorization: `Bearer ${config.get('token')}`,
+          host: `${SERVER_HOST}:${SERVER_PORT}`,
+        },
+      }
+    )
+  ).data.data;
+}
+
+async function switchOrganization(config: Configstore, organizationId: string): Promise<string> {
+  return (
+    await axios.post(
+      API_SWITCH_ORGANIZATION_FORMAT_URL.replace('{organizationId}', organizationId),
+      {},
+      {
+        headers: {
+          Authorization: `Bearer ${config.get('token')}`,
+        },
+      }
+    )
+  ).data.data;
+}
+
+async function createApplication(config: Configstore, applicationName: string): Promise<IApplication> {
+  return await axios.post(
+    API_CREATE_APPLICATION_URL,
+    { name: applicationName },
+    {
+      headers: {
+        authorization: `Bearer ${config.get('token')}`,
+        host: `${SERVER_HOST}:${SERVER_PORT}`,
+      },
+    }
+  );
+}
+
+function getOrganizationId(userJwt: string): string {
+  const tokens = userJwt.split('.');
```

Comment:
Loved the DIY here :) There is a library called decode-jwt or something like that that does validation as well on the payload. we can still use this one but what ever you think :)

Reply:
I'm am not sure if we need validation on this level. The check here is pretty simple, thanks for pointing me at the external library, I will consider the use of the library on the upcoming commit :)